Formula/phylonium.rb

```diff
@@ -0,0 +1,39 @@
+class Phylonium < Formula
+  # cite Kl_tzl_2019: "https://doi.org/10.1093/bioinformatics/btz903"
+  desc "Fast and Accurate Estimation of Evolutionary Distances"
+  homepage "https://github.com/EvolBioInf/phylonium"
+  url "https://github.com/EvolBioInf/phylonium/archive/refs/tags/v1.5.tar.gz"
+  sha256 "5cfab29b8753f5b38e6363fa1ba387578863e3bbb8fd1a9722abdc32d1c90686"
+  license "GPL-3.0-or-later"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "libdivsufsort"
```

Comment:
```suggestion
  depends_on "gcc"
  depends_on "libdivsufsort"

  fails_with gcc: "4.9" # requires C++14(?)
  fails_with gcc: "5" # requires C++14(?)
```

The homebrew's default compiler gcc-5 could not build the source codes. Use the latest gcc compiler and insert `failes_with gcc: "5"` to avoid using the old ones. If you know why it doesn't work with this old compiler, please leave it in the comments.